src/server/graphql/resolvers.js

```diff
@@ -142,6 +153,25 @@ export default {
         missionId: id
       });
       return models.Mission.findByPk(id);
-    }
+    },
+    updateUser: async (_, { input }, { currentUser }) => {
```

Comment:
so I think the way I would have done it is having updateUser { name, email, whatever}